sample/allofhands_hawaii2016/event

```diff
@@ -0,0 +1,90 @@
+{
+	"id": 3,
+	"name": "All Hands/2016 Hawaii",
+	"latitude": null,
+	"longitude": null,
+	"location_name": "Hilton Waikoloa village, Hawaii",
```

Comment:
@mahikaw 
I guess a sample must be as comprehensive as possible. Try to add the lat-long fields for them